### DISCUSSION TITLE
Zemismart Roller Shade Motor not working after being flashed

### DISCUSSION THREAD

**Discussion by chriswalker01** [arendst/Tasmota#11141] [Q&A] [answered]
Hey everyone, has anyone got Zemismart Roller Shade Motors working? https://templates.blakadder.com/zemismart_M2805EIGB.html

I was able to get Tasmota on the USB bridge over serial but can't control the blind. I'm hoping Zemismart haven't changed something to make Tasmota incompatible since the template was uploaded.

I tried following this https://tasmota.github.io/docs/Configuration-Procedure-for-New-Devices/ , but none of the relays caused the blind to move.
![image](https://user-images.githubusercontent.com/14160438/109372713-1d63b780-78ff-11eb-95b5-a01d4bfd182a.png)
![image](https://user-images.githubusercontent.com/14160438/109372749-5308a080-78ff-11eb-92d4-3936ad92c5bd.png)
![image](https://user-images.githubusercontent.com/14160438/109372752-556afa80-78ff-11eb-9a47-abe624d4eff4.png)

Any ideas?

**Comment by ascillato2** [comment, marked as answer]
Please, address this to the [Tasmota Support Chat](https://discord.gg/Ks2Kzd4). The chat is a better and more dynamic channel for helping you.

---------------------------

### Support Information 
See [Docs](https://tasmota.github.io/docs/) for more information.
See [Chat](https://discord.gg/Ks2Kzd4) for more user experience.
See [Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
